shap/__init__.py

```diff
@@ -1,22 +1,21 @@
-# flake8: noqa
 
 __version__ = "0.42.1"
 
-from ._explanation import Explanation, Cohorts
-
 # explainers
+
+from ._explanation import Cohorts, Explanation
```

Comment:
small nit, but Explanation and Cohorts aren't explainers, I'll prefer to keep it where it was